ios-xcframework/podspec.erb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+Pod::Spec.new do |s|
+  s.name = 'Gutenberg'
+  s.summary = 'Printing since 1440'
+  s.homepage = 'https://github.com/wordpress-mobile/gutenberg-mobile'
+  s.authors = 'Automattic'
+
+  s.version = '<%= version %>'
+
+  s.license = { type: '<%= license %>' }
+
+  s.ios.deployment_target = '<%= ios_version %>'
+  s.swift_version = '<%= swift_version %>'
+
+  s.source = {
+    http: "https://d2twmm2nzpx3bg.cloudfront.net/Gutenberg-<%= id %>.tar.gz"
```

Comment:
```suggestion
    http: "https://cdn.a8c-ci.services/Gutenberg-<%= id %>.tar.gz"
```

Fun little polish 🙂

Reply:
Excellent! I wasn't aware of that URL. Much neater 😄